frontend/ui/widget/textboxwidget.lua

```diff
@@ -55,6 +87,10 @@ function TextBoxWidget:init()
     if self.height == nil then
         self:_renderText(1, #self.vertical_string_list)
     else
+        -- luajit may segfault if we were provided with a negative height
```

Comment:
Seems like a good sanity check anyway, but segfault? Hm. :-P

Reply:
just try it: add `self.height = -10` after that comment fix and you should get like I do: 
```Backtrace:
/lib/x86_64-linux-gnu/libc.so.6(+0x84378)[0x7f0c130b6378]
./luajit(+0x1fd17)[0x55bfe2283d17]
./luajit(+0x22eb7)[0x55bfe2286eb7]
./luajit(+0x23a66)[0x55bfe2287a66]
```
I didn't spent much time investigating why that happened...

Reply:
Nice. :-D